general_rules/ExternalComputing.tex

```diff
@@ -26,6 +26,37 @@ \section{External computing}\label{rule:robot_external_computing}
 
 \textbf{Remark:} Teams are allowed to use their own software in the external computing devices (not only cloud services). This software must be publicly available to other teams for scientific purposes (evaluation, test, and benchmarking), as well as for TC for inspection. Although open-sourcing the software is not mandatory, this practice is advised and encouraged by the league.
 
+\subsection{On-Site External Computing Devices}
+\label{rule:robot_external_computing_devices}
+
+\begin{itemize}
+  \item \textbf{Location:} All external computing devices, approved by the \iaterm{Technical Committee}{TC}
+  		(see~\refsec{rule:robot_external_computing}), must be placed at the designated location, e.g., a table close
+        to the arena, referred hereinafter as the \iaterm{\textbf{E}xternal \textbf{C}omputing \textbf{R}esource
+        \textbf{A}rea} {ECRA}. The OC will announce the ECRA during setup days. A switch connected to the arena
+        wireless network will be available to teams.
+  \item \textbf{Procedure:} Before a teams' test run, external devices must be setup in time.
+  		Immediately after a teams' test slot, all equipment must be removed from the ECRA in order to give ensuing
+        teams adequate space and time to setup their devices. As soon as the referee indicates the teams' test slot
+        starts, team members are strictly forbidden to touch their external devices until the current test run is over.
+        Breaking this ``hands-off'' rule is penalized with -25 points for the corresponding test run. Also, not removing
```

Comment:
Replace ``hands-off'' with \quotes{hands-off} or \enquote{hands-off} (If csquote package is not there, add it).